README.md

```diff
@@ -22,8 +22,10 @@ are contained in the 'res' directory.
 The following programs and/or libaries need to be installed before compiling
 and installing the RMA:
 
-Asterisk (>= version 13.0.0)
-libcurl
+- Asterisk (>= version 13.0.0)
+- libcurl
+- pjproject
+- res_rtp_asterisk
```

Comment:
It's highly unlikely someone wouldn't have that module installed as part of an Asterisk install. It's possible for someone to remove it from the Asterisk installation of course, but not without taking some additional steps that depart from a typical Asterisk install.